dvc/commands/experiments/utils.py

```diff
@@ -0,0 +1,35 @@
+def add_rev_selection_flags(
```

Comment:
You can move these to [`dvc/commands/experiments/__init__.py`](https://github.com/iterative/dvc/blob/47d10c4a8b9bdc18dec17a9380f0593e43383c4c/dvc/commands/experiments/__init__.py) and just import it as:
```python
from . import add_rev_selection_flags
```

Reply:
Yes but looks like that would cause a circular import, so we must place this import inside `add_parser`.